### PR TITLE
History filter radio buttons

### PR DESCRIPTION
**Pull request by karolko**
Fixes https://github.com/api3dao/api3-dao-dashboard/issues/145

### CODE REVIEW

File: src/pages/history/history.tsx [hunk 1]
```diff
@@ -24,16 +24,25 @@ const History = () => {
...
   // TODO: Implement pagination for history proposals
   useHistoryProposals();
   useTreasuryAndDelegation();
 
-  const applyHistoryFilter = (type: ProposalType) => {
-    // TODO: I find these tab logic confusing, maybe reiterate? See https://api3workspace.slack.com/archives/C020RCCC3EJ/p1622624192006000
-    if (!proposalType || proposalType === type) history.replace(`/history?type=${type}`);
-    else history.replace(`/history`);
-  };
+  // useEffect is used because useState behaves asynchronously using onClick handler
```
Comment (Siegrift) [inline src/pages/history/history.tsx:34]: :1st_place_medal:

**Comment by Siegrift** [comment]
Resolve conflicts, but LGTM otherwise

**Review by Siegrift — DISMISSED** [review]

File: src/components/radio-button/radio-button.tsx [hunk 2]
```diff
@@ -19,11 +22,22 @@ const RadioButton = ({ label, onChange, checked, name, color }: Props) => {
...
         })}
       >
         {label}
-        <input className={styles.radioButtonInput} type="radio" onChange={onChange} checked={checked} name={name} />
-        <span className={styles.radioButtonCheckmark} />
+        <input
+          className={styles.radioButtonInput}
+          type="radio"
+          onClick={onClick}
+          onChange={onChange}
```
Comment (Siegrift) [inline src/components/radio-button/radio-button.tsx:33]: Isn't `onChange` enough?

Comment (karolko) [reply src/components/radio-button/radio-button.tsx]: unnecessary `onClick` is removed

**Review by Siegrift — APPROVED** [review]
:100: good job!

**Pull request merged**